game_patch/object/entity.cpp

```diff
@@ -201,6 +203,85 @@ CodeInjection entity_process_pre_hide_riot_shield_injection{
     },
 };
 
+// avoids gibbing if gore level is too low or if this specific corpse shouldn't gib
+CodeInjection corpse_damage_patch{
+    0x00417C6A,
```

Comment:
I believe you skip too much code. Previously it was despawning corpse when their life decreased below 0. Now they won't unless you have gibs enabled. So they are basically indestructible. Was it intentional? You can skip ` entity_blood_throw_gibs` call only. WDYT?

Reply:
It was intentional. I don't know that it makes much sense to have corpses disappear without any animation/effect/anything if gibbing is disabled based on gore level, and it seemed to me like the original code only deleted the corpse _because_ it was expecting it to have gibbed (because it only cares about explosive damage).

Perhaps my thinking on this is flawed though, hm. If you have a second to check, could you see if the code from the PS2 debug build is any different for this? (like if it maybe uses gore level to determine what to do?) Probably the original intended Volition behaviour for this should be replicated here, whatever it may be.